en/test_and_ci/continous_integration.md

```diff
@@ -1,3 +1,98 @@
+  Detects system level bugs in close, and also verifies compatibility between MAVROS and PX4.
+- `checks.yml`:
+  Runs various make targets (using `make` command).
+  The commands for each targets are documented in the [Makefile](https://github.com/PX4/PX4-Autopilot/blob/main/Makefile)
+  - `check_format`: Checks the coding style format using Clang Tidy ?
+  - `airframe_metadata`: Builds airframe XML metadata
+  - `parameters_metadata`: Builds parameter XML metadata
+  - `module_documentation`: Builds module documentation
+  - `tests`: Runs unit tests
+  - `tests_coverage`: Runs coverage tests
+  - ... (other auxiliary tests)
+- `ekf_functional_change_indicator.yml`: Check if there's any functionality change in EKF ?
+- `ekf_update_change_indicator.yml`: Check if there's any functionality change in EKF ?
+- `failsafe_sim.yml`: Builds [failsafe simulator](../config/safety_simulation.md)
+- `clang-tidy.yml`: Runs Clang Tidy to check if coding style is consistent
+- `compile_linux.yml`: Builds linux flight controller board targets
+
+## PX4 Releases/main
+
+A number of additional artifacts are built when changes are made to the `main` branch or to `release/*` branches (any releases).
+These are then deployed to other repositories, uploaded to servers, and generally made available to other systems and services that use them.
+
+Release/main actions are managed by the [.github/workflows/deploy_all.yml](https://github.com/PX4/PX4-Autopilot/blob/main/.github/workflows/deploy_all.yml) workflow.
+
+For each board target, the artifacts built are:
+
+- Parameter metadata (`parameters.*`)
+- Events metadata (`events/*.xz`)
+- Actuators metadata (`actuators.json*`)
+- Target binary (`*.px4`)
+
+These files are uploaded to the [Amazon S3 bucket server](https://px4-travis.s3.amazonaws.com/).
+
+In addition, if there are changes to the `main` branch Jenkins builds / deploys metadata used by QGroundControl and documentation builds, based on the definitions in [PX4-Autopilot/Jenkinsfile](https://github.com/PX4/PX4-Autopilot/blob/main/Jenkinsfile).
+
+
+## PX4 Documentation
+
+Jenkins uses the [PX4BuildBot](https://github.com/PX4BuildBot) account to upload the following metadata to the [PX4-user_guide](https://github.com/PX4/PX4-user_guide) repo.
+
+- `airframes_reference.md`: Markdown for [Airframes Reference Documentation](../airframes/airframe_reference.md)
+- `parameter_reference.md`: Markdown for [Parameter Reference Documentation](../advanced_config/parameter_reference.md)
+- `modules/*.md`: Markdown for [Modules documentation](../modules/README.md)
+- `graph_*.json`: JSON defining uORB Graph, visualized in [uORB Graph](../middleware/uorb_graph.md) page
+- `msg_docs/*.md`: Markdown for [uORB Messages documentation](../msg_docs/README.md)
+- `failsafe_sim/*`: Files for [Failsafe Simulator Interface](../config/safety_simulation.md)
+
+An example commit can be found [here](https://github.com/PX4/PX4-user_guide/commit/a4ffaf5fc32df8650da614aa3129b3b90794584f).
```

Comment:
Shouldn't we mention that this is done by the https://github.com/PX4/PX4-Autopilot/blob/d6b523b574875a9f640620d1e90c8277fa13781c/Jenkinsfile#L152-L191?

Due to the sentence below, I think people might confuse the Jenkins within the doc with the Jenkins by PX4 (which is the one used for this case for documentation metadata update) @hamishwillee

Reply:
Also, we can mention that it only gets triggered when the `main` branch gets altered.

Reply:
I think it would be pretty hard to be confused. However I have further re-written it to be even more clear.